src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Performance/BasicPreferHashDataOverComputeHash.Fixer.vb

```diff
@@ -71,21 +71,45 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Performance
                         SyntaxFactory.IdentifierName(hashTypeName),
                         SyntaxFactory.Token(SyntaxKind.DotToken),
                     SyntaxFactory.IdentifierName(PreferHashDataOverComputeHashAnalyzer.HashDataMethodName))
-                        Return SyntaxFactory.InvocationExpression(hashData, argumentList)
+                        Dim arg = argumentList.Arguments(0)
+                        If arg.IsNamed Then
+                            arg = DirectCast(arg, SimpleArgumentSyntax).WithNameColonEquals(SyntaxFactory.NameColonEquals(SyntaxFactory.IdentifierName("source")))
+                        End If
+                        Dim args = SyntaxFactory.ArgumentList(SyntaxFactory.SingletonSeparatedList(Of ArgumentSyntax)(arg))
+                        Return SyntaxFactory.InvocationExpression(hashData, args)
                     Case PreferHashDataOverComputeHashAnalyzer.ComputeType.ComputeHashSection
+                        Dim list = argumentList.Arguments.ToList()
+                        Dim firstArg = list.Find(Function(a) a.IsNamed = False OrElse DirectCast(a, SimpleArgumentSyntax).NameColonEquals.Name.Identifier.Text.Equals("buffer", StringComparison.Ordinal))
```

Comment:
VB is case insensitive. I think this should be `OrdinalIgnoreCase`.

Reply:
Yes, just tested. This has to be `OrdinalIgnoreCase`.